### PR TITLE
doc: describe in fuzzing.md how to reproduce a CI crash

### PR DESCRIPTION
**Pull request by jonatack**
Not sure if this is 100% accurate or missing any pertinent info, but I misremembered how to do this today and it seems like useful information to provide.

### CODE REVIEW

**Comment by laanwj** [comment]
Concept ACK, this is definitely required information if we keep running the fuzz corpus in CI.

File: doc/fuzzing.md [hunk 1]
```diff
@@ -81,6 +81,15 @@ INFO: seed corpus: files: 991 min: 1b max: 1858b total: 288291b rss: 150Mb
 …
 ```
 
+To reproduce a fuzzer crash found by the CI:
```
Comment (MarcoFalke) [inline doc/fuzzing.md:84]:
```suggestion
## Reproduce a fuzzer crash found by the CI
```

**Review by maflcko — APPROVED** [review]
ACK

File: doc/fuzzing.md (hunk 1 above)
Comment (jonatack) [reply doc/fuzzing.md]: done
```diff
-To reproduce a fuzzer crash found by the CI:
+## Reproduce a fuzzer crash found by the CI
 
 - `cd` into the `qa-assets` directory and update it with `git pull qa-assets`
 - locate the crash case described in the CI output, e.g. `Test unit written to
@@ -90,6 +90,8 @@ To reproduce a fuzzer crash found by the CI:
   `FUZZ=process_message src/test/fuzz/fuzz
   qa-assets/fuzz_seed_corpus/process_message/1bc91feec9fc00b107d97dc225a9f2cdaa078eb6`
 
+## Submit improved coverage
```

File: doc/fuzzing.md [hunk 2]
```diff
@@ -81,6 +81,17 @@ INFO: seed corpus: files: 991 min: 1b max: 1858b total: 288291b rss: 150Mb
 …
 ```
 
+## Reproduce a fuzzer crash found by the CI
```
Comment (MarcoFalke) [inline doc/fuzzing.md:84]: After a private discussion. "found" might confuse some readers, as the CI currently does not *search* for new inputs. It is repeating the existing test inputs like the unit tests.

**Review by maflcko — APPROVED** [review]
ACK

(unrelated suggestion: Could move the section header "fuzzing corpora" to the line before "The project's collection of seed corpo...")

File: doc/fuzzing.md (hunk 2 above)
Comment (jonatack) [reply doc/fuzzing.md:84]: I hesitated on "found" as well..."reported"?

**Comment by jonatack** [comment]
How about
```diff
-## Fuzzing harnesses, fuzzing output and fuzzing corpora
+## Fuzzing harnesses and output
 
 [`process_message`](https://github.com/bitcoin/bitcoin/blob/master/src/test/fuzz/process_message.cpp) is a fuzzing harness for the [`ProcessMessage(...)` function (`net_processing`)](https://github.com/bitcoin/bitcoin/blob/master/src/net_processing.cpp). The available fuzzing harnesses are found in [`src/test/fuzz/`](https://github.com/bitcoin/bitcoin/tree/master/src/test/fuzz).
 
@@ -64,6 +64,8 @@ block^@M-^?M-^?M-^?M-^?M-^?nM-^?M-^?
 
 In this case the fuzzer managed to create a `block` message which when passed to `ProcessMessage(...)` increased coverage.
 
+## Fuzzing corpora
+
 The project's collection of seed corpora is found in the [`bitcoin-core/qa-assets`](https://github.com/bitcoin-core/qa-assets) repo.

```

**Comment by jonatack** [comment]
@MarcoFalke I added a bullet point from your IRC comment.

- make sure to compile with all sanitizers, if they are needed (fuzzing runs more slowly with sanitizers enabled, but a crash should be reproducible very quickly from a crash case)

**Comment by practicalswift** [comment]
Concept ACK

Thanks for documenting this!

Will review.

**Comment by practicalswift** [comment]
ACK d8f1ea7227260e51c340271fc9a43866799b8ac4

**Pull request merged**